src/components/TopBar.js

```diff
@@ -107,10 +115,69 @@ export default Utils.connectAtom(authStore, 'authState')(class TopBar extends Co
       onClick: () => this.hideNav()
     }, [
       div({ style: styles.nav.icon }, [
-        icon(iconName, { className: 'is-solid', size: 24 })
+        icon(iconName, {
+          className: 'is-solid',
+          size: 24
+        })
       ]),
       label
     ])
+
+    const enabledCredits = h(Clickable, {
+      style: {
+        ...styles.nav.item,
+        borderBottom: 'none'
+      },
+      hover: { backgroundColor: colors.darkBlue[1] },
+      onClick: () => this.setState({ openFreeCreditsModal: true })
+    }, [
+      div({ style: styles.nav.icon }, [
+        icon('cloud', {
+          className: 'is-solid',
+          size: 20
+        })
+      ]),
+      'Sign up for $300 in free cloud credits'
```

Comment:
Thinking about how to shorten this as well... I don't want it to sound too click-bait-y but maybe something like "Click for $300 in credits" or "Sign up for free credits". 

In general I recommend keeping the nomenclature the same for the Free Credits Program. The banner and documentation always refer to it as "Free Credits", and not the "Free Cloud Credits".